src/chainparams.cpp

```diff
@@ -84,6 +84,8 @@ class CMainParams : public CChainParams {
         consensus.nSuperblockStartBlock = 600000; // TODO, the block at which 12.1 goes live.
         consensus.nSuperblockCycle = 16616; // ~(60*24*30)/2.6, actual number of blocks per month is 200700 / 12 = 16725
         consensus.nGovernanceMinQuorum = 10;
+        consensus.nGovernanceFilterElements = 100;
+        consensus.nGovernanceFilterElements = 20000;
```

Comment:
2 declarations for main, 1 for regtest and none for testnet?

main = 20000
test = 500
regtest = 100 

is this what you wanted?